karma.conf.js

```diff
@@ -11,7 +11,6 @@ module.exports = function (config) {
             'browserify'
         ],
         files: [
-            'node_modules/@babel/polyfill/dist/polyfill.js', // For async/await in tests
```

Comment:
Good! This was wrong.

Reply:
This was correct because the polyfills are used for the test files itself. Re-added it and explicitly excluded the un-minified web3 package and explicitly included the minified file provided by us.